app/models/middleware_datasource.rb

```diff
@@ -0,0 +1,8 @@
+class MiddlewareDatasource < ApplicationRecord
+  include ReportableMixin
+
+  belongs_to :ext_management_system, :foreign_key => "ems_id"
+  belongs_to :middleware_server, :foreign_key => "server_id"
+  serialize :properties
+  acts_as_miq_taggable
```

Comment:
I would move this to line 3 because it belongs more to including code than to definition of relationships. but it's a nit.